src/ports/postgres/modules/deep_learning/test/madlib_keras_predict.sql_in

```diff
@@ -85,7 +85,7 @@ SELECT assert(estimated_y IN (0,1),
 FROM cifar10_predict;
 
 DROP TABLE IF EXISTS cifar10_predict;
-SELECT assert(trap_error($TRAP$madlib_keras_predict(
+SELECT assert(trap_error($TRAP$SELECT madlib_keras_predict(
```

Comment:
This is also failing for the wrong reason i.e. wrong column names. 
`id` should be replaced by `buffer_id` and `x` by `independent_var`. 
Also once you make these changes and run this test without the trap_error function, the failure is not meaningful 
```
ERROR:  plpy.SPIError: function array_upper(bytea, integer) does not exist
LINE 2:                 SELECT array_upper(independent_var, 1) AS n_...
                               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
QUERY:
                SELECT array_upper(independent_var, 1) AS n_0, array_upper(independent_var, 2) AS n_1, array_upper(independent_var, 3) AS n_2
                FROM cifar_10_sample_batched
                LIMIT 1
```
But that should be taken care of in a future PR.